templates/datalab/storage-configmap.template.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
```

Comment:
This looks good but ideally want these values to somehow be configured in the config.yaml, I'm not sure if in bara like Helm we can have some logic in these templates, but for example for the current production deployment we will have to fork this file to specify only glusterfs currently as we don't have NFS deployed, not too much of a problem currently if it's going to be complex though.

Reply:
As discussed, this isn't possible with Bara - and given we might replace Bara with Helm at some point, it's probably not worth the effort.